qiskit/transpiler/passes/layout/sabre_layout.py

```diff
@@ -370,3 +383,13 @@ def _compose_layouts(self, initial_layout, pass_final_layout, qregs):
         qubit_map = Layout.combine_into_edge_map(initial_layout, trivial_layout)
         final_layout = {v: pass_final_layout._v2p[qubit_map[v]] for v in initial_layout._v2p}
         return Layout(final_layout)
+
+
+@dataclasses.dataclass
+class _DisjointComponent:
+    dag: DAGCircuit
+    coupling_map: CouplingMap
+    initial_layout: NLayout
+    final_permutation: "list[int]"
+    sabre_result: "SabreResult"
```

Comment:
Should this be `(SwapMap, ndarray, NodeBlogResult)` instead of `SabreResult`?

Reply:
Heh NodeBlog ✍️ 😄

Reply:
Done in ff33d8d09.